src/github/amexboy/git_ops.clj

```diff
@@ -0,0 +1,30 @@
+(ns github.amexboy.git-ops
+  (:use [github.amexboy.shell :only [sh]]))
+
+(defn git-prune-worktree [root]
+  "Removes dead worktrees"
```

Comment:
you may want to combine them in a defmulti method for all git operations
See http://clojuredocs.org/clojure.core/defmulti